entrypoint.sh

```diff
@@ -46,5 +46,15 @@ install_driftctl || log_error "Fail to install driftctl"
 qflag=""
 quiet_flag
 
+# Store scan output in variable 
+scan_output="$(driftctl scan $qflag $INPUT_ARGS)"
+
+# Escape scan output to handle multilines
+scan_output="${SCAN_OUTPUT//$'\n'/'%0A'}"
```

Comment:
It will not work since you use SCAN_OUTPUT here where it should be scan_output.

So here the scan_output will equal to empty string

Reply:
@wbeuil updated in latest commit